packages/config/src/android/EasBuild.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs-extra';
+import path from 'path';
+
+import gradleScript from './EasBuildGradleScript';
+
+function hasApplyLine(content: string, applyLine: string) {
+  return (
+    content
+      .split('\n')
+      // Check for both single and double quotes
+      .some(line => line === applyLine || line === applyLine.replace(/"/g, "'"))
+  );
+}
```

Comment:
This function is now defined in two places:
- here
- in `packages/expo-cli/src/commands/eas-build/build/builders/AndroidBuilder.ts`

I'd prefer to have it defined in one place.